src/world/http_server.cpp

```diff
@@ -18,6 +18,7 @@ along with this program.  If not, see http://www.gnu.org/licenses/
 
 ===========================================================================
 */
+
```

Comment:
I decided to keep the newlines the same as LSB to avoid future merge conflicts, even though i'd rather not add them to a PR